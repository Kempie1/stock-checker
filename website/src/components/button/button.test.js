import React from "react"
import Button from "./button"
import { render } from '@testing-library/react';
import TestRenderer from 'react-test-renderer';


test("<Button/> doesn't crash", ()=>{
    const div = document.createElement("div");
    render (<Button>test</Button>,div)
})

test("<Button/> matches snapshot", ()=>{
    const renderedComponent = TestRenderer.create(<Button>test</Button>)
    expect(renderedComponent).toMatchSnapshot()
})