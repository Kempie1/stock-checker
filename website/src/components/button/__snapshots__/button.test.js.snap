// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button/> matches snapshot 1`] = `
<button
  className="btn btn--primary btn--medium"
>
  test
</button>
`;
